### ISSUE TITLE
Soft buttons are inaccessible while playing on full screen 

### ISSUE THREAD

**Issue by SameenAhnaf** [libre-tube/LibreTube#3557]
### Steps to reproduce

1. Enable auto full screen.
2. Play a video.
3. Rotate the phone.
4. Open drop-down menu.
5. Watch the video till end.
6. Rotate the phone again.

### Expected behavior

Soft buttons should be seen at the bottom.

### Actual behavior

Soft buttons are inaccessible until the notification panel is scrolled down.

### LibreTube version

0.14.0

### Android version

Android 12

### Other details

https://user-images.githubusercontent.com/79245641/232173548-e068cace-a701-4e71-bdcb-a5b343617fd9.mp4

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Is the crash still reproducible with the latest debug build?

**Comment by jonhermansen** [comment]
Simply switching a video in and out of full screen causes the same issue for me. I do not use auto-rotate.

Only after pulling down notifications, does the 3 button navigation reappear.

Version 0.14.1

**Comment by Bnyro** [comment]
We recently had some changes related to that issue and due to no activity here for a while I adsume that issue no longer exists.
